src/components/CallToActionButton.vue

```diff
@@ -9,9 +9,8 @@
 -->
 
 <template>
-  <DestinationDataProvider v-if="action" :destination="action">
+  <DestinationDataProvider v-if="action" :destination="action" v-slot="{ url, title }">
```

Comment:
Has the Call to Action been tested with this change? I recall we had an issue with this in the past

Reply:
This is just a syntax change for the slot. The tests would have failed if its wrong :)